snet-ui/SubscribeToNotification/index.tsx

```diff
@@ -126,9 +128,9 @@ function SubscribeToNotification(
       </Box>
       <Box sx={{ display: "flex", justifyContent: "center", mt: 3,  ml: 3.3}}>
         {alertMessage.message ? (
-          <Alert severity="error" sx={{ width: "49.8%", height: "45px"}}>
-            {alertMessage.message }
-          </Alert>
+         <Alert  icon={<ErrorOutline sx={{color: '#A70147'}} />} severity="error" sx={{  fontWeight: "200", height: "45px", width: "49.5%", backgroundColor:`${colors.LIGHT_RED} !important`}}>
```

Comment:
Don't hardcode colorcode here. Define it on colors.ts file and use.

Reply:
Also, please enable VS prettier and linter so that we dont break the prettify guidelines

Reply:
@raamb I have enabled prettier and updated it sir